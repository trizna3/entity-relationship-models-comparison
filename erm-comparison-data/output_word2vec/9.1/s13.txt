marine-soldiers
transport model-EMPTY
airplane-airplanes
fight model-EMPTY
airplane model-EMPTY
rocket type-rocket types
marine role-EMPTY
GENERALIZATION_TO_11_ASSOCIATION:S-fighter planes-airplanes
CHANGE_CARDINALITY:S-rocket types-fighter planes
CHANGE_CARDINALITY:S-fighter planes-airplanes
REBIND_1NN1_TO_MN:S-fighter planes
