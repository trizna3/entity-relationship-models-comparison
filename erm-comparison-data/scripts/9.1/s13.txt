entity set airplanes {
	"serial number"
	"name"
	"model"
};

entity set "fighter planes" {
};

entity set "transport planes" {
	"person capacity"
	"material volume"
	"material weight"
};

entity set "rocket types" {
};

entity set soldiers {
	"name"
	"health"
	"maximal health"
	"speed"
	"strength"
	"agility"
	"inteligence"
};

entity set roles {
	"name"
	"health modifier"
	"maximal health modifier"
	"speed modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

association between {
	airplanes MANY
	soldiers MANY
	roles MANY
}{};

association "rocket count" between 
{
	"fighter planes" MANY
	"rocket types" MANY
}{};

generalization from "fighter planes" to "airplanes";

generalization from "transport planes" to "airplanes";
