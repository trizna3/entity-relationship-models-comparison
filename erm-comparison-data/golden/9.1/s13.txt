airplane model-model
fight model-fighter planes
rocket type-rocket types
transport model-transport planes
airplane-airplanes
marine-soldiers
marine role-roles
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-model-airplanes